### ISSUE TITLE
Fix run package on MacOS with Apple M1 cpu

### ISSUE THREAD

**Issue by pcpl2** [pcpl2/flutterMediainfo#11]
Problem has originally posted by @vadimbarda in https://github.com/pcpl2/flutterMediainfo/issues/10#issuecomment-1429842547

**Comment by vadimbarda** [comment]
@pcpl2 I just did few tests. There are results:
1 v 0.0.3 works on arm64 if it is compiled on arm64. Works on x86_64 if it is compiled on x86_64. It is not works on x86_64 if it is compiled on arm64
2 v 0.0.0.4-dev works on x86_64 if it is compiled on any platform.  It doesn't on arm64 if it is compiled on any platform

**Comment by vadimbarda** [comment]
@pcpl2 I don't think it is good idea to publish 0.0.4. I'm afraid it will not works at all for people on arm64 architecture. As for me at this moment I have ability to choose between two machines on Codemagic CI (arm64 and x86_64). Thank you

**Comment by pcpl2** [comment]
I'll wait with release 0.0.4 until i can debug what's going on with this issue on arm64

**Comment by vadimbarda** [comment]
@pcpl2 There is an article how to make universal binaries https://developer.apple.com/documentation/apple-silicon/building-a-universal-macos-binary Maybe it can help for your investigation

**Comment by pcpl2** [comment]
I'm published 0.0.4-dev.3

In this version i'm added build support library for arm64.

Unfortunately, I haven't tested if it works because I currently don't have access to the M1 computer

**Comment by vadimbarda** [comment]
@pcpl2 Thank you, but this version doesn't work for me on x86 architecture even in debug mode
`flutter: Invalid argument(s): Failed to load dynamic library '/Users/vadym/Dev/projects/media_casa/build/macos/Build/Products/Debug/MediaCasa.ai.app/Contents/Frameworks/flutter_media_info.framework/Resources/libnativeUtils.dylib': dlopen(/Users/vadym/Dev/projects/media_casa/build/macos/Build/Products/Debug/MediaCasa.ai.app/Contents/Frameworks/flutter_media_info.framework/Resources/libnativeUtils.dylib, 0x0001): tried: '/Users/vadym/Dev/projects/media_casa/build/macos/Build/Products/Debug/MediaCasa.ai.app/Contents/Frameworks/flutter_media_info.framework/Resources/libnativeUtils.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/Users/vadym/Dev/projects/media_casa/build/macos/Build/Products/Debug/MediaCasa.ai.app/Contents/Frameworks/flutter_media_info.framework/Resources/libnativeUtils.dylib' (no such file), '/Users/vadym/Dev/projects/media_casa/build/macos/Build/Products/Debug/MediaCasa.ai.app/Contents/Frameworks/flutter_media_info.framework/Resources/libnativeUtils.dylib' (no such file), '/Users/vadym/Dev/projects/media_casa/build/macos/Build/Products/Debug/MediaCasa.ai.app/Contents/Frameworks/flutter_media_info.framework/Versions/A/Resources/libnativeUtils.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/Users/vadym/Dev/projects/media_casa/build/macos/Build/Products/Debug/MediaCasa.ai.app/Contents/Frameworks/flutter_media_info.framework/Versions/A/Resources/libnativeUtils.dylib' (no such file), '/Users/vadym/Dev/projects/media_casa/build/macos/Build/Products/Debug/MediaCasa.ai.app/Contents/Frameworks/flutter_media_info.framework/Versions/A/Resources/libnativeUtils.dylib' (no such file)`

**Comment by pcpl2** [comment]
Maybe it's some cache problem because i renamed the extra library for macOS from "libnative_utils.dylib" to "libnativeUtils.dylib". Are you able to check if the file is in this location?

**Comment by vadimbarda** [comment]
@pcpl2 Thanks for watching. Instead of libnativeUtils.dylib I see libmediainfo.0.dylib in Resources folder using your version 0.0.4-dev.3
